### PR TITLE
Add "send" transfer types

### PR DESCRIPTION
**Pull request by pacmessica**

### CODE REVIEW

File: api/transaction.go [hunk 1]
```diff
@@ -474,3 +475,64 @@ func findLog(lf []LogFormat, index int) LogFormat {
 	}
 	return LogFormat{}
 }
+
+func produceTransfers(se *shared.SubsetEvent, transferType string, logf LogFormat) (err error) {
```
Comment (pacmessica) [inline api/transaction.go:479]: ok place for this? or move it

**Review by lukanus — APPROVED** [review]

File: api/transaction.go (hunk 1 above)
Comment (lukanus) [reply api/transaction.go:479]: Yeah this is fine :)

**Review by silentlight — APPROVED** [review]
Looks good. Do we have somewhere in the README types of transfers we support?

**Comment by lukanus** [comment]
@silentlight we support all, and we're usually return it in different place so that's only the extension of that, "because we're nice" after testing, if this didn't broke anything on wider range - probably tomorrow - we'll ship it and I'll add this field to Learn Docs.

**Pull request merged**